README.md

```
@@ -1,21 +1,46 @@
-![Lift Simulation Example](Lift-Simulation-Example.png "Lift Simulation Example")
 
-# Requirements
-  1. Have a page where you input the number of floors and lifts from the user
-  2. An interactive UI is generated, where we have visual depictions of lifts and buttons on floors
-  3. Upon clicking a particular button on the floor, a lift goes to that floor
 
-  Milestone 1:
-   - Data store that contains the state of your application data
-   - JS Engine that is the controller for which lift goes where
-   - Dumb UI that responds to controller's commands
+
```

markdownlint-cli2 (0.17.2)

Multiple consecutive blank lines
Expected: 1; Actual: 4

(MD012, no-multiple-blanks)